Source/WebCore/rendering/svg/SVGTextLayoutAttributes.h

```diff
@@ -46,7 +48,8 @@ class SVGTextLayoutAttributes {
     explicit SVGTextLayoutAttributes(RenderSVGInlineText&);
 
     void clear();
-    static float emptyValue();
+    static float emptyValue() { return std::numeric_limits<float>::max() - 1; }
+    static bool isEmptyValue(float value) { return std::isnan(value); }
```

Comment:
That is wrong, you partially merged the NaN-change.

Reply:
It leads to following test not rendering any text: https://web.archive.org/web/20150712020900/http://output.jsbin.com/imajow/1/quiet